.github/workflows/test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           DOPPLER_TOKEN_JOB_CREATOR: ${{ secrets.INTEGRATION_TEST_DOPPLER_TOKEN_JOB_CREATOR }}
           DOPPLER_TOKEN_SOLVER: ${{ secrets.INTEGRATION_TEST_DOPPLER_TOKEN_SOLVER }}
           WEB3_PRIVATE_KEY: ${{ secrets.INTEGRATION_TEST_WEB3_PRIVATE_KEY }}
```

Comment:
@walkah do we actually need these secrets in GitHub to run the tests as an action?  I tried removing them and saw the tests fail, then when i added them back it worked; however, on subsequent runs of the tests they ended up failing again so just want to clarify with you before leaving/removing them for good

Reply:
@walkah and I chatted about this last week. We would like to remove our dependency on GitHub secrets for tests because external contributors are not able to access them when tests run on their forks.

Here is an example from a failing run: https://github.com/Lilypad-Tech/lilypad/actions/runs/10419436102/job/28868000750#step:7:9. The Doppler tokens presumably are not set because the external contributor was unable to access them.

Reply:
Sounds good, the tests are acting flaky on this PR anyway, so i'll clean this up as I investigate what's going on